frontend/src/components/game/Gametab.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import axios from "axios";
+import Board from './Board';
+import History from './History';
+
+/* eslint-disable */
+class Gametab extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isGameMode: false,
+      difficulty: 2,
+    };
+    this.handleStartGame = this.handleStartGame.bind(this);
+    this.handleEndGame = this.handleEndGame.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.submitScore = this.submitScore.bind(this);
+  }
+  
+  // enable the game mode
+  handleStartGame() {
+    this.setState({
+      isGameMode: true,
+    });
+  }
+  
+  // disable the game mode
+  handleEndGame() {
+    this.setState({
+      isGameMode: false,
+    });
+  }
+  
+  // toggle the difficulty based on user input
+  handleChange(event) {
+    var d = 2;
+    if (event.target.value == "Easy") {
+      d = 2;
+    } else if (event.target.value == "Medium") {
+      d = 3;
+    } else {
+      d = 4;
+    }
+    this.setState ({
+      difficulty: d,
+    });
+  }
+
+  // test() {
+  //   const item = { score: 60 }
+  //   axios
+  //     .post("/api/game/", item)
+  //     .catch((err) => console.log(err));
+  // }
+  
+  // render the gametab, create the history and board components
+  render() {
+    return (
+      <div className="game">
+      {!this.state.isGameMode &&
+        <div className="game-init">
```

Comment:
please indent this
in the unlikely event that they actually go through our code, i dont wanna lose points for something that can be avoided with an auto-formatter that runs on file save
i think you can also run `npm run lint:fix` to auto format all jsx files
thats kinda extreme though, so you might want to run `eslint --fix` on individual files

Reply:
or maybe it was `npm run prettify:fix`? i forget which one does what